adala/environments/kafka.py

```diff
@@ -55,14 +55,16 @@ async def initialize(self):
             self.kafka_input_topic,
             bootstrap_servers=self.kafka_bootstrap_servers,
             value_deserializer=lambda v: json.loads(v.decode("utf-8")),
+            enable_auto_commit=False,
```

Comment:
Should this also be added to the output topic consumer?

Reply:
also, mind leaving a comment about what this + acks=all does?

Reply:
Yeah I will add to output as well. For whatever reason, maybe due to rate of messages sent to output topic, we never hit this issue there. Will leave comments as well yes